bap-builder/DockerMode.go

```
@@ -1,74 +1,71 @@
 package main
 
 import (
+	"bringauto/modules/bringauto_log"
 	"bringauto/modules/bringauto_docker"
-	"fmt"
-	"os"
 	"path"
 )
 
 // BuildDockerImage
 // process Docker mode of cmd line
 //
 func BuildDockerImage(cmdLine *BuildImageCmdLineArgs, contextPath string) error {
+	contextManager := ContextManager{
+		ContextPath: contextPath,
+	}
 	buildAll := cmdLine.All
 	if *buildAll {
-		return buildAllDockerImages(contextPath)
+		return buildAllDockerImages(contextManager)
+	}
+
+	dockerfilePath, err := contextManager.GetImageDockerfilePath(*cmdLine.Name)
+	if err != nil {
+		return err
 	}
-	return buildSingleDockerImage(contextPath, *cmdLine.Name)
+	buildSingleDockerImage(*cmdLine.Name, dockerfilePath)
```

Why the error is ignored?

---

Not ignored now.